self-hosting-novu/kubernetes.mdx

```diff
@@ -3,23 +3,24 @@ title: 'Kubernetes'
 description: 'Learn how to deploy Novu with K8s'
 icon: 'text-height'
 ---
-Kubernetes is an open-source platform that automates the deployment, scaling, and management of containerized applications. 
-It helps manage containers, ensuring they run reliably across machines. 
-Key terms include pods (containers), deployments (scaling), services (networking), and more. 
-Kubernetes simplifies complex tasks, making it easier to develop and run applications consistently in various environments.
-Learn more about [Kubernetes](https://kubernetes.io/docs/home/)
 
-# 🏗️ This page is still in creation process
+<Warning>
+    We do not plan to support any official public kubernetes deployment at this time.
+    This guide is to help assist advanced users who want to deploy Novu on their own kubernetes cluster.
+</Warning>
 
-We believe in the power of collaboration. If you have insights to share or would like to contribute to the development of this page, we welcome your input. To get involved, follow these steps:
+Kubernetes is an open-source platform that automates the deployment, scaling, and management of containerized applications.
 
-**Open an Issue:** If you've identified an area for improvement or have a suggestion, start by opening an issue in our repository. This allows us to track and discuss the proposed changes.
+You can find a community driven [helm deployment](https://github.com/novuhq/novu/blob/next/docker/kubernetes/helm/README.md) in our repository.
 
-**Create a Pull Request (PR):** Once you're ready to contribute, fork our repository and create a branch. Implement your changes within this branch and submit a pull request to our **Staging branch**.
+## Web Container Image
 
-By following these steps, you're helping us enhance the quality and depth of our resources. We appreciate your effort in making this page an even more valuable asset.
+Due to the way we build our web container image,
+you will need to build your own image and push it to your own registry.
+This is due to us compiling the frontend when the container starts.
```

Comment:
Are there any steps to help them build their own image we can add here? @Cliftonz

Reply:
@unicodeveloper Added more detail but unfortunately I would have to do the whole process to add all of the details.
We can open a issue for the community though, thoughts?

Reply:
This works. Sure. I'll merge this in.